test/test_datasets_download.py

```diff
@@ -391,6 +391,25 @@ def widerface():
     )
 
 
+def kinetics():
+    return itertools.chain(
+        *[
+            collect_download_configs(
+                lambda: datasets.Kinetics(
+                    path.join(ROOT, "Kinetics", split),
```

Comment:
@bjuncek I don't see any reason why the user needs to manually set the last part of the path to the correct split. This should be handled internally.

```suggestion
                    ROOT,
```

Reply:
makes sense; it was done to avoid breaking BC with the API, but we can do this more elegantly.
Please check my subsequent commit for that.